app/views/groups/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1 class="page-title"> Transactions for <%= @group.name %> Category</h1>
+
+<div class="transactions-list">
+  <% @entities.each do |entity| %>
+    <div class="transaction-item">
+      <div class="transaction-info">
+        <h3 class="entity-name"><%= link_to entity.name, group_entity_path(@group, entity) %></h3>
+        <p class="entity-amount">$<%= entity.amount %></p>
+      </div>
+      <div class="transaction-actions">
+        <%= button_to 'Delete', group_entity_path(@group, entity), method: :delete, data: { confirm: 'Are you sure you want to delete this transaction?' }, class: 'delete-button' %>
+      </div>
+    </div>
+  <% end %>
+</div>
+
+<%= link_to 'Add a New Transaction', new_group_entity_path(@group), class: 'add-transaction-button' %>
+<%= link_to 'Back', groups_path, class: 'back-link' %>
```

Comment:
Please note that your design is a bit off the original design, kindly fix it for a better UI.
Your design |  Original design
![Screenshot from 2023-08-09 19-14-47](https://github.com/adeola003/Budget_App/assets/44624138/133e2656-5439-4da2-92d6-05762171267e)  |  ![Screenshot from 2023-08-09 19-14-04](https://github.com/adeola003/Budget_App/assets/44624138/b0392c2b-21f6-4b5f-95e5-d52ce13c57e9)
Also, the total transaction can be seen at the top of the page on the original page, kindly implement this 👍🏽

Reply:
Please ensure your transaction details design matches the original Behance page design, including adding the Icon and arrow button to go back. This will make your app more user-friendly and it give a better UI 😉.

Your design | Original design
![Screenshot from 2023-08-09 19-25-42](https://github.com/adeola003/Budget_App/assets/44624138/ff1c70c5-13cc-4878-bcf9-9bf3f2d9f394)  |  ![Screenshot from 2023-08-09 19-24-42](https://github.com/adeola003/Budget_App/assets/44624138/e5d51308-7f5a-41c2-bbb5-6572cc3c9561)